node/builder_chain.go

```diff
@@ -163,7 +163,8 @@ var ChainNode = Options(
 	),
 
 	If(build.IsF3Enabled(),
-		Override(new(manifest.ManifestProvider), lf3.NewManifestProvider),
+		Override(new(*manifest.Manifest), lf3.NewManifest),
```

Comment:
This will make it easier to override the manifest while testing without changing everything.